src/zap_hooks/helpers/utilities.py

```diff
@@ -0,0 +1,44 @@
+from sys import exit
```

Comment:
for consistency - let's use underscore for the compound file names
blindxss -> blind_xss
browserstorage -> browser_storage
requestheaders -> request_headers